selfdrive/car/chrysler/chryslercan.py

```diff
@@ -53,28 +45,26 @@ def calc_checksum(data):
 def make_can_msg(addr, dat):
   return [addr, 0, dat, 0]
 
-def create_lkas_heartbit(car_fingerprint):
-  # LKAS_HEARTBIT (729) Lane-keeping heartbeat.
-  msg = '0000000820'.decode('hex')  # 2017
-  return make_can_msg(0x2d9, msg)
+def create_lkas_heartbit(packer, lkas_status_ok):
```

Comment:
I think this message can be simply forwarded in panda, rigtht?

Reply:
Sure thing. Implemented and tested in a car that it works.